src/watertap_contrib/reflo/costing/watertap_reflo_costing_package.py

```diff
@@ -168,6 +168,18 @@ def add_specific_thermal_energy_consumption(
             specific_thermal_energy_consumption_constraint,
         )
 
+    def add_LCOW(self, flow_rate, name="LCOW"):
+        super().add_LCOW(flow_rate, name=name)
+        # NOTE: Add reference to flow rate here so that if a user tries to add_LCOW
+        # on a REFLOSystemCosting block after add_LCOW is called on TreatmentCosting,
+        # the parameter can be renamed from LCOW to LCOT and use the same flow rate.
+        add_object_reference(self, "_LCOW_flow_rate", flow_rate)
+
+    def add_LCOT(self, flow_rate):
```

Comment:
Why do we need this separate function? You can just use add_LCOW with name of "LCOT" and point to relevant flowrate, right?